bundles/org.openhab.ui.basic/src/main/resources/snippets/input.html

```diff
@@ -0,0 +1,26 @@
+<div class="mdl-form__row mdl-cell mdl-cell--6-col mdl-cell--8-col-tablet %visibility_class%">
+	<span class="mdl-form__icon">
+		<img data-icon="%category%" src="../icon/%category%?state=%state%&format=%icon_type%&anyFormat=true" />
```

Comment:
Please replace this line by `%icon_snippet% (recent change on all other snippet).
`

Reply:
Done